crates/aptos/src/common/types.rs

```diff
@@ -1090,6 +1125,14 @@ pub struct MovePackageDir {
     /// Select bytecode, language, compiler for Move 2
     #[clap(long)]
     pub move_2: bool,
+
+    /// Select optimization level
+    #[clap(long, value_parser = clap::value_parser!(OptimizationLevel))]
+    pub optimization_level: Option<OptimizationLevel>,
+
+    /// Enable lint
+    #[clap(long)]
+    pub lint: bool,
```

Comment:
Do these apply to non-move2?

Reply:
No.  Good point, I'll add a warning about OptimizationLevel.  There's already one for `aptos move lint` with V1.